content/build-tool.md

```diff
@@ -79,7 +79,16 @@ One difference between pre-published packages and local app packages is that the
 
 <h4 id="npm-adding">Adding packages to your app</h4>
 
-Meteor 1.3 will have seamless integration with NPM, and you will be able to simply `npm install` these packages into your app directory. Until then, the easiest way to use NPM packages in your app is [`meteorhacks:npm`](https://atmospherejs.com/meteorhacks/npm).
+To install an NPM package into your app, you can simply run NPM's standard install command:
+
+```
+npm install --save-dev <package name>
+```
+
+XXX: Is it `meteor npm`
+XXX: Do we use --save-dev
+XXX: What about creating an initial `package.json`
```

Comment:
This would definitely make sense in the universe where a `meteor create` gives you an app with NPM dependencies. If we have `meteor create --react` or similar, that will be necessary, and I bet a Meteor 1.4 app will require NPM dependencies for some core Meteor stuff?